### PR TITLE
FSeq2-NMS

### PR DESCRIPTION
**Pull request by charsyme**
Implementation of the FSeq2-NMS method (for person detection only) using SSD as detector.

### CODE REVIEW

**Comment by charsyme** [comment]
This PR is ready for review.

File: src/opendr/perception/object_detection_2d/nms/fseq2_nms/algorithm/__pycache__/__init__.cpython-38.pyc
Comment (tsampazk) [inline src/opendr/perception/object_detection_2d/nms/fseq2_nms/algorithm/__pycache__/__init__.cpython-38.pyc:1]: I think this file should be removed from the remote repo as they are [marked in .gitignore](https://github.com/opendr-eu/opendr/blob/b3d6ce670cdf63469fc5766630eb295d67b3d788/.gitignore#L19)

File: src/opendr/perception/object_detection_2d/nms/fseq2_nms/algorithm/__pycache__/fseq2_model.cpython-38.pyc
Comment (tsampazk) [inline src/opendr/perception/object_detection_2d/nms/fseq2_nms/algorithm/__pycache__/fseq2_model.cpython-38.pyc:1]: I think this file should be removed from the remote repo as they are [marked in .gitignore](https://github.com/opendr-eu/opendr/blob/b3d6ce670cdf63469fc5766630eb295d67b3d788/.gitignore#L19)

**Review by tsampazk — CHANGES_REQUESTED** [review]
Thank you @charsyme  I have some comments, most are minor inconsistencies between the docs and code and some other details.

**Comment by tsampazk** [comment]
@charsyme Please double-check if the suggestions are valid and batch commit them through the github UI for conveniency.

File: src/opendr/perception/object_detection_2d/nms/fseq2_nms/algorithm/__pycache__/fseq2_model.cpython-38.pyc
Comment (charsyme) [reply src/opendr/perception/object_detection_2d/nms/fseq2_nms/algorithm/__pycache__/fseq2_model.cpython-38.pyc:1]: Done

File: src/opendr/perception/object_detection_2d/nms/fseq2_nms/fseq2_nms_learner.py
[the remark this answers is not in the document]
Comment (charsyme) [reply src/opendr/perception/object_detection_2d/nms/fseq2_nms/fseq2_nms_learner.py:1]: I re-ordered them, and change some of them to private (except from init_model). I did also the same for seq2seq_nms_learner.py.

**Comment by charsyme** [comment]
@tsampazk I think I dealt with all the suggestions/comments. Sorry for some extra separate commits.

**Review by tsampazk — APPROVED** [review]
Thanks @charsyme, looks good to me! :ok_hand:

**Review by passalis — APPROVED** [review]
Thank you!

**Pull request merged**